### DISCUSSION TITLE
Welcome to redis-developer.github.io Discussions!

### DISCUSSION THREAD

**Discussion by ajeetraina** [redis-developer/redis-developer.github.io#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ajeetraina** [comment]
The GitHub action now shows successful.

**Comment by ajeetraina** [comment]
@skonaads I merged it but still see the old page. Any idea? Wonder if it is sitting in  different branch altogether?

**Comment by skonaads** [comment]
I see you've been working on it - any luck yet? It is working locally for
us - I can have a dev try to help troubleshoot.

<https://www.skona.com/>

marc nixon

lead web developer

+1 415 494 2854 x212 <+1+415+494+2854+x212>

499 Carolina St. SF, CA 94107

skona.com <https://www.skona.com/>

**Reply by tgrall** [reply]
I have "merged and forced" your design into our repo.

I have integrated all the content, and links and will continue to do it so.

Can you please refork and start from our master to be sure we are up to date on both sides.

I have also:

* removed the search form
* enable the switch to dark mode button (the css does not support that see issue #20 )

**Comment by skonaads** [comment]
Thanks Tug! Good call on the dark mode - I will make a note to add some
styles for that.
